### PR TITLE
Make the OIDC_RSA_PRIVATE_KEY setting base64

### LINKED ISSUE #1993: Base64 encode OIDC_RSA_PRIVATE_KEY
### Describe the bug
Make sure no newlines are needed in the environment variable by base64 encoding

### PR DESCRIPTION
**Pull request by se-bastiaan**
Closes #1993 

### Summary
Make the OIDC_RSA_PRIVATE_KEY setting a value that must be provided in Base64.

### How to test
Steps to test the changes you made:
1. Provide the setting in non-base64
2. Error
3. Provide in Base64
4. Works

### CODE REVIEW

File: website/thaliawebsite/settings.py [hunk 1]
```diff
@@ -514,9 +514,13 @@ def from_env(
 CORS_URLS_REGEX = r"^/(?:api/v1|api/v2|user/oauth)/.*"
 
 # OAuth configuration
+OIDC_RSA_PRIVATE_KEY = from_env("OIDC_RSA_PRIVATE_KEY", testing=None)
+if OIDC_RSA_PRIVATE_KEY is not None:
+    OIDC_RSA_PRIVATE_KEY = base64.urlsafe_b64decode(OIDC_RSA_PRIVATE_KEY)
```
Comment (pingiun) [inline website/thaliawebsite/settings.py:519]:
```suggestion
    OIDC_RSA_PRIVATE_KEY = base64.b64decode(OIDC_RSA_PRIVATE_KEY)
```

This is easier to create as it's used by the `base64` command line program

Comment (pingiun) [reply website/thaliawebsite/settings.py]: You can change this back because it's better to have it in line with the rest of the base64 decodes

**Review by JobDoesburg — DISMISSED** [review]

File: website/thaliawebsite/settings.py [hunk 2]
```diff
@@ -514,9 +514,13 @@ def from_env(
 CORS_URLS_REGEX = r"^/(?:api/v1|api/v2|user/oauth)/.*"
 
 # OAuth configuration
+OIDC_RSA_PRIVATE_KEY = from_env("OIDC_RSA_PRIVATE_KEY", testing=None)
+if OIDC_RSA_PRIVATE_KEY is not None:
+    OIDC_RSA_PRIVATE_KEY = base64.b64decode(OIDC_RSA_PRIVATE_KEY)
```
Comment (se-bastiaan) [inline website/thaliawebsite/settings.py:519]:
```suggestion
    OIDC_RSA_PRIVATE_KEY = base64.urlsafe_b64decode(OIDC_RSA_PRIVATE_KEY)
```

File: website/thaliawebsite/settings.py (hunk 1 above)
Comment (se-bastiaan) [reply website/thaliawebsite/settings.py:519]: Done!

**Review by pingiun — APPROVED** [review]

**Pull request merged**